jans-cedarling/cedarling_opa/convert.py

```
@@ -0,0 +1,226 @@
+        raise ValueError(f"policy_store is missing required fields: {missing}")
+
+    return metadata
+
+
+def aggregate_json_directory(dir_path: Path) -> _t.Dict:
+    """Reads all .json files in a directory and aggregates them into a single dict."""
+    aggregated = {}
+    if not dir_path.exists():
+        return aggregated
+
+    for object in dir_path.iterdir():
```

Ruff (0.15.21)

[error] Variable `object` is shadowing a Python builtin

(A001)